src/fsevents/fsevents_stubs.c

```diff
@@ -209,21 +209,17 @@ CAMLprim value dune_fsevents_stop(value v_t) {
   CAMLreturn(Val_unit);
 }
 
-static inline value Val_some(value v) {
-  CAMLparam1(v);
-  CAMLlocal1(some);
-  some = caml_alloc_small(1, 0);
-  Field(some, 0) = v;
-  CAMLreturn(some);
-}
-
 CAMLprim value dune_fsevents_runloop_get(value v_t) {
   CAMLparam1(v_t);
   dune_fsevents_t *t = (dune_fsevents_t *)Nativeint_val(v_t);
   if (t->runloop == NULL) {
     CAMLreturn(Val_int(0));
   } else {
-    CAMLreturn(Val_some(caml_copy_nativeint((intnat)t->runloop)));
+    CAMLlocal2(v_some, v_runloop);
```

Comment:
Having the `CAMLlocal2` in a block is somewhat unorthodox, but I guess it works?

Reply:
If you think it's suspicious I can move it to the top.

Reply:
Sure, let's move it to the top! Thanks.